src/mmu.c

```diff
@@ -25,20 +25,28 @@ static const uint8_t bootrom[256] = {
     0xF5, 0x06, 0x19, 0x78, 0x86, 0x23, 0x05, 0x20, 0xFB, 0x86, 0x20, 0xFE, 0x3E, 0x01, 0xE0, 0x50
 };
 
-ErrorCode mem_init(void) {
-    uint8_t *rom_bytes = get_rom_bytes();
-
+ErrorCode mmu_init(void) {
     mem = calloc(1, 0x10000);
+    uint8_t *rom_bytes = get_rom_bytes();
+    if (rom_bytes == NULL) {
+        LOG_ERROR("ROM-bytes buffer is NULL!\n");
+    } else {
+        memcpy(&mem[0x0000], &rom_bytes[0x0000], 0x4000);
+        memcpy(&mem[0x4000], &rom_bytes[0x4000], 0x4000);
```

Comment:
With the frequency `0x4000` and so on are used, there should be defines for that (or at least a static const variable)

Reply:
I introduced some defines for values already in #25. Except for these two memcpy lines

Reply:
Did you have a reason to skip those lines in particular?

Reply:
`&rom_bytes[0x0000]` is equivalent to just `rom_bytes`

Reply:
![image](https://user-images.githubusercontent.com/20692341/196654250-8a2eeed3-4842-4d52-8eb4-c9bb21326076.png)

Reply:
Ah nvm, I suspect you used `&rom_bytes[0x0000]` for continuity reasons. Didn't spot the `0x4000` in the last line.